docs/design/datacontracts/Loader.md

```diff
@@ -38,6 +38,9 @@ TargetPointer GetLoaderAllocator(ModuleHandle handle);
 TargetPointer GetThunkHeap(ModuleHandle handle);
 TargetPointer GetILBase(ModuleHandle handle);
 ModuleLookupTables GetLookupTables(ModuleHandle handle);
+TargetPointer GetModuleLookupMapElement(TargetPointer table, uint rid, out TargetNUInt flags);
+bool IsCollectibleLoaderAllocator(ModuleHandle handle);
```

Comment:
I think we'll want to have a LoaderAllocator contract. There is a bunch of ... stuff... on there, and I feel like we'll likely need something there.